app/src/main/Member.js

```diff
@@ -823,6 +824,27 @@ export default class Member extends CoreMember {
         });
     }
 
+    /**
+     * Populate customer tracking metadata headers in token calls.
+     *
+     * @param {CustomerTrackingMetadata} customerTrackingMetadata
+     * @return new member
+     */
+    addCustomerTrackingMetadata(
+        customerTrackingMetadata: CustomerTrackingMetadata,
+    ): Member {
+        return Util.callSync(this.addCustomerTrackingMetadata, () => {
+            const newMember = new Member(this._options);
```

Comment:
do we need to create a new member?
`addCustomerTrackingMetadata` sounds more like changing an existing member

Reply:
removed creation of a new member